src/api/app/views/models/_project.xml.builder

```diff
@@ -14,6 +14,7 @@ xml.project(project_attributes) do
     end
   end
 
+  xml.url(my_model.url) unless my_model.url.blank?
```

Comment:
Please use ``present?`` instead 
http://apidock.com/rails/Object/present%3F
(However, would be fine in sticking to ``blank?`` as the following lines also use ``blank?``)